src/helpers/figures/charts/runtime/chart_data_extractor.ts

```diff
@@ -94,6 +97,124 @@ export function getBarChartData(
   };
 }
 
+function getDateTimeLabel(value: number, stamp: Granularity): string {
+  switch (stamp) {
+    case "day_of_week": {
+      return DAYS[value - 1].toString();
+    }
+    case "hour_number": {
+      return getHoursLabel(value);
+    }
+    case "month_number": {
+      return MONTHS[value - 1].toString();
+    }
+    case "iso_week_number": {
+      return `W${value < 10 ? "0" : ""}${value}`;
+    }
+    case "quarter_number": {
+      return `Q${value}`;
+    }
+    default: {
+      return value.toString();
+    }
+  }
+}
+
+function getHoursLabel(hour: number): string {
+  return hour < 12 ? `${hour} AM` : `${hour - 12} PM`;
```

Comment:
Should be translated

Reply:
'AM' and 'PM' ?

Reply:
Not exactly AM and PM, but the date formatting. Would be worth to check what Luxon does